modules/mobile/spec/request/observations_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/mr_client_helpers'
+require 'medical_records/client'
+require_relative '../support/helpers/iam_session_helper'
+
+RSpec.describe 'observations', type: :request do
+  include MedicalRecords::ClientHelpers
+  include SchemaMatchers
+  # include JsonSchemaMatchers
+
+  let(:user_id) { '32000225' }
+  let(:va_patient) { true }
+  let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
```

Comment:
should this be using the new SIS user stuff that Kris made?

Reply:
Done